misc/fibonacci_formula.c

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file
+ * @brief [finding Fibonacci number of any number n using binet's closed form formula](https://en.wikipedia.org/wiki/Fibonacci_number#Binet's_formula)
```

Comment:
```suggestion
 * @brief Finding Fibonacci number of any `n` number using [Binet's closed form formula](https://en.wikipedia.org/wiki/Fibonacci_number#Binet's_formula)
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Inelegant